.gitconfig

```diff
@@ -0,0 +1,7 @@
+# Add the following to your .gitconfig file.
+[diff]
```

Comment:
Thanks for the contribution. Could you remove the modification to `.gitconfig` which is common for all developers. They are likely best applied to your local git global configuration via `git config --global ...`